frontend/components/pages/LearningRecordList.tsx

```diff
@@ -0,0 +1,156 @@
+'use client'
+import {
+  Box,
+  Card,
+  CardBody,
+  CardHeader,
+  Center,
+  Flex,
+  FormControl,
+  HStack,
+  Heading,
+  Input,
+  SimpleGrid,
+  Text,
+  VStack,
+} from '@chakra-ui/react'
+import { LearningRecordType } from '../../types'
+import { useCustomToast } from '../../hooks/useCustomToast'
+import { PRIMARY_FONT_COLOR } from '../../constants'
+
+type LearningRecordTypeProps = {
+  learningRecords: LearningRecordType[]
+}
+
+const formatDate = (dateString) => {
+  const date = new Date(dateString)
+  const year = date.getFullYear()
+  const month = date.getMonth() + 1
+  const day = date.getDate()
+  return `${year}年${month}月${day}日`
+}
+
+const formatLearningTime = (minutes) => {
+  const hours = Math.floor(minutes / 60)
+  const remainingMinutes = minutes % 60
+  return `${hours}時間${remainingMinutes}分`
+}
+
+export function LearningRecordList({
+  learningRecords,
+}: LearningRecordTypeProps) {
+  const { showErrorToast } = useCustomToast()
+
+  if (!Array.isArray(learningRecords)) {
+    showErrorToast('学習記録の取得に失敗しました')
+    return null
+  }
+
+  const groupedRecords: Record<string, LearningRecordType[]> = {}
+  learningRecords.forEach((record) => {
+    if (!groupedRecords[record.video_id]) {
+      groupedRecords[record.video_id] = []
+    }
+    groupedRecords[record.video_id].push(record)
+  })
+
+  const totalLearningTime: Record<string, number> = {}
+  Object.keys(groupedRecords).forEach((videoId) => {
+    totalLearningTime[videoId] = groupedRecords[videoId].reduce(
+      (total, record) => total + record.learningTime,
+      0,
+    )
+  })
+  const totalTime = Object.values(totalLearningTime).reduce(
+    (total, time) => total + time,
+    0,
+  )
+
+  return (
+    <Center bg={'gray.200'}>
+      <VStack mx="auto" padding={'60px 96px'}>
+        {Object.keys(groupedRecords).length === 0 ? (
+          <Heading color={PRIMARY_FONT_COLOR} fontSize="36px">
+            学習記録はありません
+          </Heading>
+        ) : (
+          <FormControl>
```

Comment:
```suggestion
          <Box>
```

がいいような気がします（formではないので）